tools/appliance/systemvmtemplate/template-base_x86_64-target_aarch64.json

```diff
@@ -0,0 +1,91 @@
+{
+  "_license": "Apache License 2.0",
+  "builders": [
+    {
+      "boot_command": [
+        "c<wait>",
+        "linux /install.a64/vmlinuz <wait>",
+        "preseed/url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/preseed_aarch64.cfg <wait>",
+        "debian-installer=en_US.UTF-8 <wait>",
+        "auto <wait>",
+        "language=en locale=en_US.UTF-8 <wait>",
+        "kbd-chooser/method=us <wait>",
+        "keyboard-configuration/xkb-keymap=us <wait>",
+        "netcfg/get_hostname=systemvm <wait>",
+        "netcfg/get_domain=apache.org <wait>",
+        "country=IN keymap=us <wait>",
+        "fb=false <wait>",
+        "debconf/frontend=noninteractive <wait>",
+        "console-setup/ask_detect=false <wait>",
+        "console-keymaps-at/keymap=us <wait>",
+        "---",
+        "<enter><wait>",
+        "initrd /install.a64/initrd.gz",
+        "<enter><wait>",
+        "boot<enter><wait>"
+      ],
+      "boot_wait": "60s",
+      "disk_interface": "virtio",
+      "cdrom_interface": "virtio-scsi",
+      "disk_size": "5000M",
+      "format": "qcow2",
+      "headless": true,
+      "http_directory": "http",
+      "iso_checksum": "sha512:4980197fa30646a43df2672d98500a182d6636422e39e809d986551356f4d24b1ffab6d745331369b27f2e9a5326f020ba141df5772ac2c6328c08cce73de7e8",
+      "iso_url": "https://cdimage.debian.org/mirror/cdimage/archive/11.10.0/arm64/iso-cd/debian-11.10.0-arm64-netinst.iso",
+      "net_device": "virtio-net",
+      "output_directory": "../dist",
+      "qemu_binary": "qemu-system-aarch64",
+      "qemuargs": [
+        [
+          "-m",
+          "500M"
+        ],
+        [
+          "-machine",
+          "virt"
+        ],
+        [
+          "-cpu",
+          "cortex-a72"
+        ],
+        [
+          "-smp",
+          "1"
+        ],
+        [ "-bios", "/usr/share/qemu-efi-aarch64/QEMU_EFI.fd" ],
```

Comment:
@StepBee I've testing this change - instead of pflash flag, with this bios flag at least the machine boots up; the build is slow but then works!